bee-common/README.md

```diff
@@ -0,0 +1 @@
+# bee-common
```

Comment:
Reasoning was that until an actual release the README would change too much. I'll add one sentence so that it's not empty but I'll write the actual README when we release.